physical/mysql/mysql.go

```diff
@@ -193,6 +170,89 @@ func NewMySQLBackend(conf map[string]string, logger log.Logger) (physical.Backen
 	return m, nil
 }
 
+func NewMySQLClient(conf map[string]string, logger log.Logger) (*sql.DB, error) {
+	var err error
+  
+  // Get the MySQL credentials to perform read/write operations.
+	username, ok := conf["username"]
+	if !ok || username == "" {
+		return nil, fmt.Errorf("missing username")
+	}
+	password, ok := conf["password"]
+	if !ok || password == "" {
+		return nil, fmt.Errorf("missing password")
+	}
+
+	// Get or set MySQL server address. Defaults to localhost and default port(3306)
+	address, ok := conf["address"]
+	if !ok {
```

Comment:
Looks like this block doesn't have a closing bracket?

Reply:
ha thanks should have looked here a few minutes ago while I was getting fun errors from go fmt like this.

```
mschuett-C02TC2VYGTDY:vault schuettm$ go fmt ./physical/mysql/mysql.go
physical/mysql/mysql.go:257:37: missing ',' in argument list
physical/mysql/mysql.go:259:2: missing ',' in argument list
physical/mysql/mysql.go:260:3: expected operand, found 'return'
physical/mysql/mysql.go:261:3: missing ',' before newline in argument list
physical/mysql/mysql.go:262:21: expected '==', found '='
physical/mysql/mysql.go:263:2: expected operand, found 'return'
physical/mysql/mysql.go:264:1: expected operand, found '}'
physical/mysql/mysql.go:268:2: missing ',' in argument list
physical/mysql/mysql.go:270:24: missing ',' before newline in argument list
physical/mysql/mysql.go:271:2: expected operand, found 'defer'
physical/mysql/mysql.go:273:9: missing ',' in argument list
exit status 2
```